gcb/build/cloudbuild.yaml

```diff
@@ -19,21 +19,21 @@ steps:
   - ${_RELEASE_TOOL_REPO}
   waitFor: [ '-' ]
 
-- name: 'gcr.io/$PROJECT_ID/k8s-cloud-builder'
+- name: 'gcr.io/k8s-staging-release-test/k8s-cloud-builder:v1.13.8-1'
```

Comment:
Had a short look at gcbmgr but I’m still not sure: can we use `${_KUBE_CROSS_VERSION}` here as well instead of the hard coded version?

Reply:
@saschagrunert -- It's hardcoded here because setting `_KUBE_CROSS_VERSION` today is dependent on the bash libraries and I don't want to introduce another thing that's dependent on them.

Reply:
Makes sense, see another question from my side: https://github.com/kubernetes/release/pull/1141/files#discussion_r386089975